Add `Verified users` (#36)
osuTitanic/wiki: wiki/People/Verified_Users/en.md

> # Verified Users

Reviewer: ```suggestion
# Verified users
```

> Verified Users are well-known/top players who have verified their identity.

Reviewer: ```suggestion
Verified users are top players or content creators who have verified their identity.
```

> ## Current Verified Users

Reviewer: ```suggestion
## Current Verified users
```

User | Bancho Account
---|:---:

> [TheShadowOfDark](https://osu.titanic.sh/u/64)  | [TheShadowOfDark](https://osu.ppy.sh/users/5795337)

Reviewer: ```suggestion
[TheShadowOfDark](https://osu.titanic.sh/u/64) | [TheShadowOfDark](https://osu.ppy.sh/users/5795337)
```

> [index](https://osu.titanic.sh/u/82)            | [Dawns](https://osu.ppy.sh/users/4639477)

Reviewer: ```suggestion
[index](https://osu.titanic.sh/u/82) | [Dawns](https://osu.ppy.sh/users/4639477)
```

> [Spare](https://osu.titanic.sh/u/92)            | [Spare](https://osu.ppy.sh/users/2204373)

Reviewer: ```suggestion
[Spare](https://osu.titanic.sh/u/92) | [Spare](https://osu.ppy.sh/users/2204373)
```

> [Champ](https://osu.titanic.sh/u/96)            | [The Champion](https://osu.ppy.sh/users/1719471)

Reviewer: ```suggestion
[Champ](https://osu.titanic.sh/u/96) | [The Champion](https://osu.ppy.sh/users/1719471)
```

> [Plasma](https://osu.titanic.sh/u/191)          | [Plasma](https://osu.ppy.sh/users/10077431)

Reviewer: ```suggestion
[Plasma](https://osu.titanic.sh/u/191) | [Plasma](https://osu.ppy.sh/users/10077431)
```

> [mrekk](https://osu.titanic.sh/u/208)           | [mrekk](https://osu.ppy.sh/users/7562902)

Reviewer: ```suggestion
[mrekk](https://osu.titanic.sh/u/208) | [mrekk](https://osu.ppy.sh/users/7562902)
```
